hpat/hiframes/pd_series_ext.py

```diff
@@ -482,13 +482,6 @@ def resolve_sort_values(self, ary, args, kws):
         out = SeriesType(ary.dtype, ary.data, out_index)
         return signature(out, *args)
 
-    @bound_function("array.take")
```

Comment:
This is signature definition of the function (like declaration in C). Need to find and remove implementation of the algorithm (like definition in C)

Reply:
As I see implementation is provided by Numba.

Reply:
Could you please point me Numba code there this method implemented?

Reply:
https://github.com/numba/numba/blob/c9b4355446e630301f2551fbd032dd56bec9ebce/numba/targets/arrayobj.py#L3616-L3675

Reply:
I see many other methods for type `array` in this file. Are you sure this type is NumPy array? I don't understand why we need this signature definition here. I see it converts output from `array.take` to `pd.Series`. Do you have an idea why? Do we have failed tests if this signature removed?